.github/container/test-pax.sh

```diff
@@ -32,7 +34,7 @@ usage() {
     exit $1
 }
 
-args=$(getopt -o a:b:s:o:n:h --long additional-args:,batch-per-gpu:,dtype:,enable-te,enable-dropout,enable-fused-attn,model-type:,evaluate,steps:,help,multiprocess,output:,data-parallel:,fsdp:,tensor-parallel:,pipeline-parallel:,nodes: -- "$@")
+args=$(getopt -o a:b:s:o:n:h --long additional-args:,batch-per-gpu:,dtype:,enable-te,enable-dropout,enable-fused-attn,model-type:,enable-fmha:,evaluate,steps:,help,multiprocess,output:,save-hlo:,data-parallel:,fsdp:,tensor-parallel:,pipeline-parallel:,nodes: -- "$@")
```

Comment:
We already have an `enable-fused-attn` flag which enables TE FMHA. Do you think we should merge `enable-fused-attn` and `enable-fmha` into a single flag?

Reply:
I don't see that flag being used, is it a dummy arg? I can remove enable-fused-attn since enable-fmha is already added in t5x test ([here](https://github.com/NVIDIA/JAX-Toolbox/blob/hmonish/add_fmha_paxml_test/.github/container/test-t5x.sh#L26)) and will be helpful in keeping the naming convention simpler.

Reply:
It's used here: https://github.com/NVIDIA/JAX-Toolbox/blob/f44cdef645698a4f0d28ad34b7d100f48ee7da11/.github/container/test-pax.sh#L90-L93 Since these flags essentially do the same thing, but one is intended to be used with TE and one is used without TE, I think it would make sense to merge the two and just use this flag in conjunction with `enable-te` to determine whether to set `NVTE_FUSED_ATTN` or `--xla_gpu_enable_cudnn_fmha`

Reply:
Oh, I see but how to handle the scenario (if it's a feasible one) where user wants to enable TE with NVTE_FUSED_ATTN but not --xla_gpu_enable_cudnn_fmha ?

Reply:
I was thinking that if `--enable-te` and `--enable-fmha` are both set, we would set `NVTE_FUSED_ATTN=1`. If just `--enable-fmha` is set, we would instead use `--xla_gpu_enable_cudnn_fmha`. When using TE, I think `--xla_gpu_enable_cudnn_fmha` should be a no-op. Will confirm with the TE team.

Reply:
I was thinking that if `--enable-te` and `--enable-fmha` are both set, we would set `NVTE_FUSED_ATTN=1`. If just `--enable-fmha` is set, we would instead use `--xla_gpu_enable_cudnn_fmha`. When using TE, I think `--xla_gpu_enable_cudnn_fmha` should be a no-op. Will confirm with the TE team.

Reply:
Confirmed that `--xla_gpu_enable_cudnn_fmha` is a no-op when TE is enabled

Reply:
sounds good, lemme merge the flags then.